README.md

```diff
@@ -22,7 +22,7 @@ const settings = {
       {
         test: /\.css$/,
         exclude: /node_modules/,
-        loader: 'typed-css-modules'
+        loader: 'typed-css-modules-loader'
```

Comment:
Webpack allows to specify loaders without `-loader` suffix. See https://webpack.github.io/docs/using-loaders.html#referencing-loaders